data.md

```diff
@@ -26,41 +27,33 @@ Files can be browsed by:
-
-The data can be downloaded via FTP, Aspera and Globus GridFTP. More information about using Aspera or Globus can be found in our FAQ.
-
-[How to download files using Aspera](/faq/how-download-files-using-aspera)  
-[How to download files using Globus](/faq/can-i-access-1000-genomes-data-globus-online)
+# Download data in bulk
 
-####FTP Hierarchy
+We recommend downloading IGSR data via Globus ([instructions](https://www.internationalgenome.org/faq/can-i-access-1000-genomes-data-globus-online)). Data can also be downloaded via FTP ([FTP site](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/)) or Aspera ([instructions](https://www.internationalgenome.org/faq/how-download-files-using-aspera)). FTP access is subject to rate limiting, so it is essential to download local copies for your work rather than try to access via range queries remotely.
 
-The FTP structure was changed in September 2015. The revised structure is described in the [FTP site structure README](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/README_ftp_site_structure.md). 
+Data is organised by collection and includes data shared pre-publication,  intermediate and working data as well as analysis files described in publications. A set of README files provides additional information.
 
-####Other data sources
 
-During the main 1000 Genomes Project, the NCBI acted as a mirror of the EBI hosted 1000 Genomes Project FTP site and also uploaded alignments and variant calls to an Amazon S3 bucket. This mirroring process stopped in September 2015. The NCBI FTP site and the Amazon S3 bucket still host 1000 Genomes Project data but no longer mirror new data. Both these locations reflect the structure of the FTP site in August 2015 and hold all the pilot, phase 1 and phase 3 data. NCBI and Amazon do not hold new alignments based on GRCh38, the current reference genome.
+#### 1000 Genomes Project - other data locations
 
-NCBI FTP Site : [ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp](ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp)  
-Amazon S3 : [s3://1000genomes](denied:s3://1000genomes)
+Data from the original 1000 Genomes Project (pilot, phase 1 and phase 3 data) are also available at the following sites.
 
-Information on Amazon Web Services can be found on 1000 Genomes [public data set page](http://aws.amazon.com/datasets/4383) or directly on [http://s3.amazonaws.com/1000genomes](http://s3.amazonaws.com/1000genomes).
+* DDBJ: [https://ftp.ddbj.nig.ac.jp/mirror_database/1000genomes/](https://ftp.ddbj.nig.ac.jp/mirror_database/1000genomes/)
+* NCBI: [https://ftp-trace.ncbi.nih.gov/1000genomes/ftp/](https://ftp-trace.ncbi.nih.gov/1000genomes/ftp/)
+* AWS: [https://registry.opendata.aws/1000-genomes/](https://registry.opendata.aws/1000-genomes/)
```

Comment:
Also AWS and Amazon Web services are pointing to the same data and are the same thing, suggesting we keep only one

Reply:
Good spot!